### DISCUSSION TITLE
Is setting up more than the default Cases status options available?

### DISCUSSION THREAD

**Discussion by beaaron** [Security-Onion-Solutions/securityonion#7895] [Unsupported Versions] [answered]
This is not a highly technical question. Our team just needs to know if it is possible to add other status' besides "Open", "In-progress", "Closed" in Cases. If so how do you perform the actions needed to edit more options in the status drop down menu?

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/soc-customization.html#cases:
> [Cases](https://docs.securityonion.net/en/2.3/cases.html#cases) comes with presets for things like category, severity, TLP, PAP, and status. You can modify these presets by copying the appropriate presets file from /opt/so/saltstack/default/salt/soc/files/soc/ to /opt/so/saltstack/local/salt/soc/files/soc/, making changes there, and then restarting SOC.

**Comment by beaaron** [comment]
Thank you Mr. Burks. I found it in the documentation an hour or so after
posting. I appreciate the response, have a great day good sir.

-Aaron
ᐧ
